server/desktop-api.js

```diff
@@ -54,10 +53,109 @@ function serveTelemetryAPI (app) {
     })
   })
 
-  var basicAuth = auth(secret.credentials)
-  var fileServer = express.static(TELEMETRY_PATH)
-  var indexServer = serveIndex(TELEMETRY_PATH)
-  app.use('/desktop/telemetry', [basicAuth, indexServer, fileServer])
+  app.use('/desktop/telemetry/', [
+    auth(secret.credentials),
+    serveTelemetryDashboard,
+    express.static(TELEMETRY_PATH),
+    express.static(path.join(__dirname, '../static/telemetry-dashboard'))
+  ])
+}
+
+// Summarize telemetry information: active users, monthly growth, most common errors, etc
+function serveTelemetryDashboard (req, res, next) {
+  var isSummary = false
+  if (req.url === '/') {
+    var path = req.originalUrl // eg '/desktop/telemetry'
+    if (!path.endsWith('/')) return res.redirect(path + '/')
+  } else if (req.url === '/summary') {
+    isSummary = true
+  } else {
+    return next()
```

Comment:
?

It will fall thru to the static handlers, and then if those don't match it will 404, right?